zenduty/assets/service_checks.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Since there are no Service checks for this integration, this file can be removed